src/components/CatalogueHarmonizationPanel.vue

```diff
@@ -60,19 +83,29 @@
         if (this.harmonizationTableData[cohort] === undefined) {
           return 0
         }
-        return this.harmonizationTableData[cohort].length
+        return this.harmonizationTableData[cohort].filter(harmonization => harmonization.status !== 'zna').length
       },
 
       isVariableHarmonized (variable) {
-        return this.harmonizedVariables.includes(variable)
+        return this.harmonizedVariables.find((harmonization) => {
+          return harmonization.variable === variable
```

Comment:
If this should be a boolean result I think you're looking for [`Array.some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some)

Reply:
Thanks!